applications/ContactStructuralMechanicsApplication/custom_utilities/contact_utilities.cpp

```diff
@@ -0,0 +1,325 @@
+// KRATOS  ___|  |                   |                   |
+//       \___ \  __|  __| |   |  __| __| |   |  __| _` | |
+//             | |   |    |   | (    |   |   | |   (   | |
+//       _____/ \__|_|   \__,_|\___|\__|\__,_|_|  \__,_|_| MECHANICS
+//
+//  License:             BSD License
+//                                       license: StructuralMechanicsApplication/license.txt
+//
+//  Main authors:    Vicente Mataix Ferrandiz
+//
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "utilities/openmp_utils.h"
+#include "utilities/math_utils.h"
+#include "includes/model_part.h"
+#include "custom_utilities/contact_utilities.h"
+
+namespace Kratos
+{
+namespace ContactUtilities
+{
```

Comment:
We had this discussion not long ago in the KTC and we decided to avoid creating new namespaces. Instead we thought about the possibility of creating a few generic and limited ones. [Discussion outcomes here](https://github.com/KratosMultiphysics/Kratos/issues/7434).

Reply:
Done